microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode.md

```diff
@@ -0,0 +1,192 @@
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 03/21/2022
+---
+# Simplify DevTools using Focus Mode
+
+Focus Mode is a new user interface for DevTools. It is designed to reduce the clutter and complexity of DevTools without compromising its feature set.
+
+Focus Mode replaces the main row of tabs with an **Activity Bar**, a compact toolbar with distinctive icons. The **Activity Bar** makes it possible to pin, rearrange, and open favorite tools for quick access. It also allows access to user settings, help, and other features.
+
+Focus Mode also provides a **Quick View** list to open a second tool alongside the tool already selected in the **Activity Bar**.
+
+![Screenshot of DevTools, undocked in its own window, with the Focus Mode experiment](media/focus-mode.png)
+
+
+<!-- ====================================================================== -->
+## Enable Focus Mode
+
+The Microsoft Edge team is enabling Focus Mode to a subset of users to collect early feedback. Some users will have Focus Mode enabled by default, while others need to enable it first.
+
+To check if you have Focus Mode enabled already, or to enable or disable it:
+
+1. Press `F1` to open **Settings**.
+
+1. Click **Experiments** and scroll down to the **Focus Mode** checkbox.
+
+   If the checkbox is selected, Focus Mode is already enabled. Select or clear it to enable or disable Focus Mode.
+
+    ![Screenshot of the Settings page in DevTools, showing the Experiments tab, scrolled down to the Focus Mode checkbox](media/focus-mode-pref.png)
+
+1. Close the **Settings** page. If you changed the Focus Mode experiment state, click the **Reload DevTools** button for the change to take effect.
+
+
+<!-- ====================================================================== -->
+## Main differences from the default user interface
+
+Here are the main differences between the DevTools default user interface and Focus Mode.
+
+### Activity Bar
+
+The **Activity Bar** is the most visible difference from the default user interface. Use it to open tools, help features, and settings.
+
+By default, the position of the **Activity Bar** adapts to the location of the DevTools window. If DevTools is undocked or docked to the bottom of the browser, the **Activity Bar** will be horizontal. If DevTools is docked to the side of the browser, the **Activity Bar** will be vertical, and tool names will not be visible. This enables quicker tool access when screen width is limited. You can change the **Activity Bar** position in the **Customize and control DevTools** (**...**) menu as follows.
+
+Focus Mode with top aligned **Activity Bar** in DevTools docked.
+
+![Screenshot of the Focus Mode top aligned Activity Bar with DevTools docked](media/focus-mode-docked.png)
+
+Focus Mode with top aligned **Activity Bar** in DevTools pop out window.
+
+![Screenshot of the Focus Mode top aligned Activity Bar with DevTools pop out window](media/focus-mode-undocked.png)
```

Comment:
The last sentence ("You can change the position ... as follows") makes me think the screenshots are going to show how to change the Activity Bar position, but they instead show DevTools docked vs. undocked. I think it's OK to just remove that "You can change..." sentence and the first screenshot of DevTools docked. A later section describes how to change it more in depth, so I don't think it's needed here, too.

Reply:
fixing